app/views/staff/training_sessions/show.html.erb

```diff
@@ -1,12 +1,16 @@
 <section id ="page-contents" >
 
-<div>Trainer: <%= @staff.name %></div><br>
+<div>Trainer: <%= @staff.name %></div>
+<div>Course: <%= @current_training_session.course %></div><br>
 
 <div>Training Type: <%= @current_training_session.training.name %>
     <br>
     <%= form_for staff_training_session_path(@current_training_session), method: 'PATCH', action: 'update' do |f| %>
       <% label_tag 'changed_params[training_id]' %>
       <%= select_tag 'changed_params[training_id]', options_from_collection_for_select(Training.all, 'id', 'name'), prompt: 'change training' %><br>
+
+      <% label_tag 'changed_params[course]' %>
+      <%= select_tag 'changed_params[course]',  options_for_select(['GNG1105', 'GNG2205']), prompt: 'change course' %><br>
```

Comment:
I would suggest that the options for this be present in `TrainingSession` as a method.